src/lib/CollectStore.js

```diff
@@ -145,7 +145,7 @@ export default class CollectStore {
               arguments: '0 0 0 * * *',
               worker: 'konnector',
               worker_arguments: {
-                konnector: connection.konnector.attributes.slug,
+                konnector: connection.konnector.slug,
```

Comment:
We may also simply remove your last commit if throwing an Error causes too much dev for now. It is not a priority IMHO.